src/gwt/src/org/rstudio/studio/client/server/ServerErrorCause.java

```diff
@@ -36,4 +36,6 @@ public String toString()
    private final int code_;
    private final String category_;
    private final String message_;
+   
+   public static final int FILE_NOT_FOUND = 2;
```

Comment:
Is this value platform-independent? Guessing it is, but just checking.

Reply:
Indeed it is, although I think it's mostly just a happy coincidence. On Unix the associated error code ENOENT (no such path or directory) maps to 2 on Linux + macOS; on Windows Boost defines its own 'file not found' error code that just so happens to equal 2 as well. (Perhaps Boost chose the Windows error codes explicitly to match the common Unix error codes?)